tests/solidity/contracts/OpCodes.sol

```diff
@@ -193,12 +193,6 @@ contract OpCodes {
      //number
      assembly {  pop(number())}
 
-     //difficulty
-     assembly {  pop(difficulty())}
-
-     //gaslimit
-     assembly {  pop(gaslimit())}
-
```

Comment:
Op code change we discussed. Just wanted to tag this one to make sure it doesn't get mnissed.